docs/admin/kubelet-authentication-authorization.md

```diff
@@ -0,0 +1,81 @@
+---
+assignees:
+- liggitt
+
+---
+
+* TOC
+{:toc}
+
+## Summary
+
+The kubelet's HTTPS endpoint exposes APIs which give access to data of varying sensitivity,
+and allow performing operations of varying power on the node and within containers.
```

Comment:
By "varying power" do you mean "requiring stronger/weaker permissions"?

How about:
"and allow you to perform operations with varying levels of power on the node and within containers."